rebar.config.erl

```diff
@@ -68,13 +68,10 @@ project_app_dirs() ->
     ["apps/*", alternative_lib_dir() ++ "/*", "."].
 
 plugins() ->
```

Comment:
Nitpick: perhaps keep the separation (for the future information), but concat the plugin list where it's used? (And remove plugins from the test profile overlay)